libraries/enum_primitive/src/cast.rs

```diff
@@ -58,7 +58,6 @@ pub trait ToPrimitive {
     }
 
     /// Converts the value of `self` to an `u64`.
-    #[inline]
     fn to_u64(&self) -> Option<u64>;
```

Comment:
This makes me nervous for an automated change to do....?

Reply:
Well I did have to delete that line based on clippy...I don't know what difference it makes for it to be there or not be there.

https://rust-lang.github.io/rust-clippy/master/index.html#inline_fn_without_body

Reply:
Yeah, it is trait method without implementation should be fine

Reply:
Very recent versions of Rust make having this here a warning.

Reply:
I'd like to think we can trust the compiler to inline this in all cases where it's germane

Reply:
Shouldn't one use `#[inline(always)]` instead of `#[inline]`, to avoid the warning?

Reply:
@gendx no, the warning is because it's a function prototype i.e, it has no body (implementation), thus `inline` it does nothing